bsp/boards/radio.h

```diff
@@ -66,7 +66,7 @@ void     radio_setFrequency(uint8_t frequency);
 void     radio_rfOn(void);
 void     radio_rfOff(void);
 // TX
-void     radio_loadPacket(uint8_t* packet, uint8_t len);
+void     radio_loadPacket(uint8_t* packet, uint16_t len);

```

Comment:
@jmmunoz86 , it's strange you didn't get an e-mail. You can look at the bottom of this page, and you'll see a red warning from Travis that at least of the tests failed. If you click on the details, you'll see exactly what failed. The idea is hence for you to keep committing to this fork until the red warning becomes a green checkbox. It's very surprising you didn't get an e-mail. AFAICT, whenever something fails, Travis sends an e-mail to all people who have committed since the last commit on which all tests passed.